src/include/iscsi-lio-server/widgets.rb

```diff
@@ -190,13 +190,16 @@ def validateGlobal(key, event)
     #	**************** Target Auth	*******************
     # handle the status of discovery authentication, enable or disable, enforce = 1 or 0.
     def handleDiscAuthStatus()
+      return unless UI.WidgetExists(Id(:auth_none))
```

Comment:
The only real change, the rest are just because some build warnings.

Reply:
I'm not sure here, should we disable the discovery authentication when the check box is not visible? Or completely skipping it is OK?

Reply:
The main problem was that it was modified when configuring LUN ACLs, where this handling is not necessary.

In the checks I did, we respect the config it is already set. And we do not touch it if the user skip the widget. 

Will try removing all completely and doing it the first time.

Reply:
I have checked it also when it is configured the first time without entering the dialog and it basically does not touch the config but by default it is disabled.

linux-59h6:/usr/share/YaST2 # cat /etc/target/lio_setup.sh | grep enforce
echo -n 0 > /sys/kernel/config/target/iscsi/discovery_auth/enforce_discovery_auth